broker/configuration.go

```diff
@@ -24,11 +24,12 @@ import (
 
 // Configuration represents configuration of Kafka broker
 type Configuration struct {
-	Address      string     `mapstructure:"address" toml:"address"`
-	Topic        string     `mapstructure:"topic" toml:"topic"`
-	PublishTopic string     `mapstructure:"publish_topic" toml:"publish_topic"`
-	Group        string     `mapstructure:"group" toml:"group"`
-	Enabled      bool       `mapstructure:"enabled" toml:"enabled"`
-	OrgWhitelist mapset.Set `mapstructure:"org_white_list" toml:"org_white_list"`
-	SaveOffset   bool       `mapstructure:"save_offset" toml:"save_offset"`
+	Address             string     `mapstructure:"address" toml:"address"`
+	Topic               string     `mapstructure:"topic" toml:"topic"`
+	PublishTopic        string     `mapstructure:"publish_topic" toml:"publish_topic"`
+	Group               string     `mapstructure:"group" toml:"group"`
+	Enabled             bool       `mapstructure:"enabled" toml:"enabled"`
+	OrgWhitelist        mapset.Set `mapstructure:"org_white_list" toml:"org_white_list"`
+	OrgWhitelistEnabled bool       `mapstructure:"enable_org_whitelist" toml:"enable_org_whitelist"`
```

Comment:
Maybe instead of adding new flag, will be better to set `org_white_list` as empty string to indicate whitelist disabling?

Reply:
dunno, would be nice to be able to distinguish between disabled whitelist and missing environment variable (and we forgot about it several times :-)

Reply:
@bond95 I was thinking the same thing, but later went with this because of @tisnik reasons.